examples/multi-destination.ts

```diff
@@ -47,6 +47,24 @@ async function openUrl(
 	if (url.startsWith(FILE_PROTOCOL)) {
 		url = url.slice(FILE_PROTOCOL.length);
 	}
+	if (
+		platform() === 'win32' &&
+		url.toLowerCase().startsWith('\\\\.\\physicaldrive')
+	) {
+		// fs.stat will fail on \\.\PhysicalDriveN on windows
+		const device = Array.from(deviceScanner.drives.values()).find((d) => {
+			return d.device !== null && d.device.toLowerCase() === url.toLowerCase();
```

Comment:
It would be nice to store `url.toLowerCase()` to a var rather than recalculating it for every single iteration, it could also be shared with
```ts
url.toLowerCase().startsWith('\\\\.\\physicaldrive')
```
above